app/main.go

```diff
@@ -136,8 +137,13 @@ func main() {
 		SyncPeriod:               *syncPeriod,
 	})
 
-	h := web.NewHandler(config)
+	updateMetrics, err := Dial(context.Background(), config)
```

Comment:
readability structure:

the name here is misleading, Dial just returns a struct - doesn't actually update any metrics (yet).

See my comment in `metric.go` about struct naming.

Reply:
Changed to CreateMetrics().  wdyt?